scripts/apps/authoring/authoring/directives/ItemCarouselDirective.ts

```diff
@@ -255,7 +259,7 @@ export function ItemCarouselDirective(notify) {
              * @param {Object} item Item object
              */
             scope.remove = function(item) {
-                controller.updateItemAssociation(scope, null, item.fieldId).then(reorderMediaItems);
+                controller.updateItemAssociation(scope, null, item.fieldId).then(() => reorderMediaItems());
```

Comment:
what was the issue here? `reorderMediaItems` is not using `this`

Reply:
`updateItemAssociation` resolves the promise to a boolean, but `reorderMediaItems` doesn't take any arguments so I'm making sure that none are passed.

Reply:
fair enough